### PR TITLE
PMM-4309 Adding Parallel Test check

### PR DESCRIPTION
**Pull request by puneet0191**
WIP

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-366-3cbc021
client docker - perconalab/pmm-client-fb:PR-366-3cbc021
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-366-3cbc021.tar.gz

File: Jenkinsfile [hunk 1]
```diff
@@ -220,6 +220,52 @@ pipeline {
                 archiveArtifacts 'results/docker/TAG'
             }
         }
+        stage('Tests Execution') {
+            parallel {
+                stage('Test: API') {
+                    steps {
+                        script {
+                           if (env.CHANGE_URL) {
```
Comment (hors) [inline Jenkinsfile]: Are you sure this check is needed? Can we remove 278, 279 lines?

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-366-2fd977c
client docker - perconalab/pmm-client-fb:PR-366-2fd977c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-366-2fd977c.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-366-a855b8b
client docker - perconalab/pmm-client-fb:PR-366-a855b8b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-366-a855b8b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-366-a855b8b
client docker - perconalab/pmm-client-fb:PR-366-a855b8b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-366-a855b8b.tar.gz

**Pull request merged**